### DISCUSSION TITLE
I created a new page template and now my custom js wont work on it.

### DISCUSSION THREAD

**Discussion by rushishelat** [holger1411/understrap#1330] [Q&A] [answered]
I created a template-home.php file under my page-templates folder and made it my default main screen. My custom js was working on the default menu screen for understrap, however after i changed the main screen to to template-home.php the custom js wont work. Any advice on why this is happening?

**Comment by danemorgan** [comment, marked as answer]
How are you enqueueing your JS?

**Reply by rushishelat** [reply]
Yep it was an issue with the enqueuing but i just got it to work. Thanks!

**Comment by gaelgerard** [comment]
What if you copy paste code from a default template in place of your custom template code and save it? Does the custom js works as expected?

There could be many reasons to your problem. You should tell us how did you create your custom template and how you are calling it.

**Reply by rushishelat** [reply]
It was an issue with the enqueuing, but i got it. Thanks though!
